app/models/docker_builder_service.rb

```diff
@@ -60,6 +61,7 @@ def push_image(tag)
         build.docker_repo_digest = "#{project.docker_repo}@#{matches[1]}"
       end
     end
+    build.docker_repo_digest ||= result.info['RepoTags'].first
```

Comment:
Could we make that column name more generic, like `docker_reference` / `docker_repo_reference`? The reference would be a digest, for our private repo or a tag for Docker Hub. Don't know if it's worth the effort, just food for thoughts.